content/administration/maintain/mailjet_api.rst

```diff
@@ -0,0 +1,213 @@
+   :align: center
+   :alt: Check DNS record in Mailjet.
+
+After successfully setting up the domain, there is an option to :guilabel:`Authenticate this domain
+(SPF/DKIM)`. This button populates :abbr:`SPF (Sender Policy Framework)` & :abbr:`DKIM (DomainKeys
+Identified Mail) records to input into the :abbr:`DNS (Domain Name System)` provider.
+
+.. note::
+   See Mailjet's :abbr:`SPF (Sender Policy Framework)`/:abbr:`DKIM (DomainKeys Identified Mail)`
+   `documentation <https://documentation.mailjet.com/hc/en-us/articles/
+   360042412734-Authenticating-Domains-with-SPF-DKIM>`_ on setting those records.
+
+.. image:: mailjet_api/authenticate.png
+   :align: center
+   :alt: Authenticate the domain with SPF/DKIM records in Mailjet.
+
+.. _maintain/mailjet-api/odoo-setup:
+
+Set up in Odoo
+==============
+
+To complete the setup, navigate to the Odoo database. With :ref:`developer-mode` turned on, go to
+the :menuselection:`Technical Menu --> Email --> Outgoing Mail Servers`. Then, create a new
+outgoing server configuration by clicking on the :guilabel:`Create` button.
+
+Next, input the `SMTP server` (in-v3.mailjet.com), `port number` (587 or 465), and `Security
+(SSL/TLS)` that was copied earlier from the Mailjet account. They can also be found `here
+<https://app.mailjet.com/account/setup>`_. It is recommended to use SSL/TLS even though Mailjet may
+not require it.
+
+For the :guilabel:`Username`, input the :guilabel:`API KEY`. For the :guilabel:`Password`, input
+the :guilabel:`SECRET KEY` that was copied from the Mailjet account to the notepad earlier. These
+settings can be found on :menuselection:`Mailjet -->  Account Settings --> SMTP and SEND API
+Settings`.
+
+Then, if the Mailjet server is used for mass emailing, set the :guilabel:`Priority` higher than
+that of any transactional email server(s). Finally, save the settings and :guilabel:`Test the
+Connection`.
+
+.. image:: mailjet_api/server-settings.png
+   :align: center
+   :alt: Odoo outgoing email server settings.
+
+.. important::
+   In order for the notifications feature to work using Mailjet, there are three settings that need
+   to be set in Odoo.
+
+   #. The :guilabel:`From Filter` needs to be set on the server configuration. It is recommended
+      to set it as a domain and not a full email address. It should match the domain in the two
+      proceeding steps. More information can be referenced :ref:`here
+      <email_servers/outgoing-email-server>`.
+   #. The :guilabel:`mail.default.from` system parameter must have the value
+      `notifications\@yourdomain.com`.
+   #. The :guilabel:`mail.default.from_filter` system parameter must have the value
+      `yourdomain.com`
+
+.. note::
+   More on setting these parameters can be found :ref:`here <email_domain/default>`.
+
+.. important::
+   Replace `yourdomain` with the custom domain for the Odoo database. If there isn't one, then use
+   the :guilabel:`mail.catchall.domain` system parameter.
```

Comment:
I feel like this information can be included in the `.. important::` block that comes before it. Having it in a separate `.. note::` and `.. important::` block with the vague wording `these parameters` can be confusing to the reader.

Reply:
I've combined these two sections into the "important" block. Thanks for pointing this out :)